src/util/methodDeprecationByVersion/4.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+module.exports = {
```

Comment:
The plugin already has a `methodDataByVersion` files. Maybe there's a way to introduce this data there, so all the information is in the same place?
Most of the information is already there - any method that exists in `3` and doesn't exist in `4` is deprecated.
This means we'd have to add a `5.js` already, but assuming you can run the `master` version of Lodash in Node, you can probably use a script to generate a file for it automatically. You don't even need to add all the information yet (that's subject to change until `v5` is released probably). You can just use empty object for each method.
This might not include the recommendation part, but it's neater, I think.